docs/docs/react-without-jsx.md

```diff
@@ -33,16 +35,24 @@ class Hello extends React.Component {
 }
 
 ReactDOM.render(
-  React.createElement(Hello, {toWhat: 'World'}, null),
+  React.createElement(Hello, { toWhat: 'World' }, null),
   document.getElementById('root')
 );
 ```
 
+`React.createElement()` takes a component, a `props` object, and any amount of children. The given component can be in the form of a string (`React.createElement("div", null, null)`), a sub-class of `React.Component`, or a plain stateless-functional component.
+
+To learn more about the function signature and arguments, you can read [the API documentation.](https://facebook.github.io/react/docs/react-api.html#createelement)
```

Comment:
Please don't use absolute links in the docs. Make it start with `/docs` since that's what the new website will use.